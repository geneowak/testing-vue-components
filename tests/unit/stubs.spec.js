import ListComponent from '@/components/List';
import { shallowMount } from '@vue/test-utils';

const listItem = {
  template: '<li>{{ movie }}</li>',
  props: ['movie'],
};

test('ShallowMount', () => {
  const wrapper = shallowMount(ListComponent, {
    stubs: {
      ListItem: listItem,
    },
  });
  expect(wrapper).toMatchSnapshot();
});
