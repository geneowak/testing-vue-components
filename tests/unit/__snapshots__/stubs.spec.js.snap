// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShallowMount 1`] = `
<div>
  <ul>
    <li>Iron Man</li>
    <li>Avengers</li>
    <li>Infitinity War</li>
  </ul>
</div>
`;
